### PR TITLE
feat(aapd-14): add confirm-remove-users page

### PR DESCRIPTION
**Pull request by bryn-pins**
## Ticket Number

https://pins-ds.atlassian.net/browse/aapd-14

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/__tests__/unit/routes/lpa/confirm-remove-user.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		require('../../../../src/routes/lpa-dashboard/confirm-remove-user');
+	});
+
+	afterEach(() => {
+		jest.resetAllMocks();
+	});
+
+	it('should define the expected routes', () => {
+		expect(get).toHaveBeenCalledWith(
+			'/confirm-remove-user/:id',
```
Comment (chrishawldev) [inline packages/forms-web-app/__tests__/unit/routes/lpa/confirm-remove-user.test.js:18]: I mentioned this on another PR earlier, and I guess this was the point raised re the value of testing the routes. Personally I think there is very limited value in testing this as we're basically just testing the express router. Not a blocker to passing review, just a talking point.

Comment (bryn-pins) [reply packages/forms-web-app/__tests__/unit/routes/lpa/confirm-remove-user.test.js:18]: I agree and is one of the things I had mentioned on our dev standards call. 
Happy to remove once we decide that's what we're doing

File: packages/forms-web-app/__tests__/unit/views/lpa/confirm-remove-user.njk.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			);
+
+			console.log(document.body.innerHTML);
+
+			expect(document.querySelector('#main-content h1').textContent.trim()).toEqual(
+				'Confirm you want to remove this user'
+			);
+		});
+
+		it('should render the email address as expected', () => {
```
Comment (chrishawldev) [inline packages/forms-web-app/__tests__/unit/views/lpa/confirm-remove-user.njk.test.js:30]: I'm also not sure on these tests, are we just testing the nun jucks rendering engine and the gov uk templates here in places? I think the main value is in testing the the nun jucks renderer has been called with the expected values that we pass in. Once we integrate the cypress tests we might also be duplicating effort here. Again not a blocker just food for thought.

**Review by chrishawldev — APPROVED** [review]
Approving with comments

File: packages/forms-web-app/__tests__/unit/views/lpa/confirm-remove-user.njk.test.js (hunk 2 above)
Comment (bryn-pins) [reply packages/forms-web-app/__tests__/unit/views/lpa/confirm-remove-user.njk.test.js:30]: We previously were doing snapshots of the views, which I did think was pointless as you just updated the snapshots mindlessly
Charlie started adding tests like this and felt it was a little more useful.
Happy to remove going forward.
This was some of the things I wanted to raise so glad you largely agree

**Pull request merged**